### ISSUE TITLE
Using Translating Strings in Title

### ISSUE THREAD

**Issue by arorakanav11** [tnicola/ngx-joyride#16]
I am trying send the title string as : 
```
<a joyrideStep="name"
[title]="'app.transaltion.name' | translate">
```
At runtime the title  is empty!!

**Comment by tnicola** [comment]
Hi  @arorakanav11 , 
Which translation library are you using? Is it your KEY `app.transaltion.name` the right one?I see a typo in  `transaltion`. Anyway, I've tried with `@ngx-translate/core` and it works fine.

**Comment by wafimardini** [comment]
Hi There
I wasn't able to use title translation 
we do use custom pipe to sync from google
usually we use it in this way: {{'Translate what ever text' | googletranslate | async}}
[title]= "'Dashboard' | googletranslate | async "   ... show text but apply any translation

Any suggestion?
Thanks in advance

**Comment by taskiukas** [comment]
Hey, are you looking into the solution for ngx http loader? Maybe I can help you?

**Comment by tnicola** [comment]
@taskiukas No, I was looking for a solution for passing async text to the joyride step. Thanks anyway for your help!

**Comment by tnicola** [comment]
@wafimardini , @arorakanav11 This bug has been fixed in the new version: [2.2.3](https://www.npmjs.com/package/ngx-joyride/v/2.2.3)
